public/utils/constants.ts

```diff
@@ -68,6 +68,9 @@ export const ANOMALY_RESULT_INDEX = '.opendistro-anomaly-results';
 
 export const BASE_DOCS_LINK = 'https://opensearch.org/docs/monitoring-plugins';
 
+export const ISM_PLUGIN_DOC_LINE =
```

Comment:
`LINE` -> `LINK`

Reply:
Discussed with UX designer, will remove this link and show AD doc link. Will add more details on AD doc to explain user should use ISM plugin to manage custom result index.